File: src/main/java/com/mynimef/bot/containers/BotFile.java
package com.mynimef.bot.containers;

public class BotFile {
    private final String path;
    private final String description;

    public BotFile(String path, String description) {
        this.path = path;
        this.description = description;
    }

    public String path() {
        return path;
    }

    public String description() {
        return description;
    }
}

File: src/main/java/com/mynimef/bot/containers/BotFile.kt
package com.mynimef.bot.containers

import java.io.File


/**
 * File that can be attached to a message
 */
data class BotFile(

    /**
     * File to attach to message
     */
    val file: File,

    /**
     * Description of a file
     */
    val description: String? = null

)
